docs/pages/admin-guides/access-controls/device-trust/enforcing-device-trust.mdx

```diff
@@ -129,7 +129,7 @@ $ tctl edit cluster_auth_preference
 
 </TabItem>
 <TabItem label="Static Config" options="Self-Hosted">
-Edit the Auth Server's `teleport.yaml` file and restart all Auth Services:
+Edit the Auth Service instances `teleport.yaml` file and restart all Auth Service instances:
```

Comment:
I'm not sure if it's correct either. It makes sense to edit `teleport.yaml` of all Auth Service instances here, but I'm not sure if it's grammatically correct to say "Auth Service instances `teleport.yaml`.

Reply:
Yes, this sentence is awkward! Changing to "Edit the teleport.yaml files for all Auth Service instances and restart them."